cadc-access-control-server/src/main/java/ca/nrc/cadc/ac/server/GroupPersistence.java

```diff
@@ -146,12 +146,14 @@ void deleteGroup(String groupID)
      *
      * @param group The group to update.
      *
+     * @return A Group instance
+     *
      * @throws GroupNotFoundException If the group was not found.
      * @throws TransientException If an temporary, unexpected problem occurred.
      * @throws AccessControlException If the operation is not permitted.
      * @throws UserNotFoundException If owner or group members not valid users.
      */
-    void modifyGroup(Group group)
+    Group modifyGroup(Group group)
```

Comment:
What was the purpose of changing this from void to return the modified group instance? 

Also, I see that addGroup is still void so if the change was for consistency/symmetry then that should also be changed (here, in the impl, in the REST API, and in the client).